notebooks/05_operationalize/pipeline-demo/aml-pipeline-use-datebricks-as-compute-target.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>I can't tell if reviewnb is rendering this badly, but the line numbers are aligned to the line after the bold title...</p><p><br></p><p>Also the link 'Click here' goes to azure portal list of databricks resources, a better link would be here: <a href="https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-manage-workspace" target="_blank">https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-manage-workspace</a></p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>